samples/cpp/text_generation/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ endfunction()
 set (SAMPLE_LIST
     greedy_causal_lm
     encrypted_model_causal_lm
+    gguf_example
```

Comment:
I suppose we need to remove this line as well as the `samples/cpp/text_generation/gguf_example.cpp` file